src/distances.jl

```diff
@@ -24,53 +32,76 @@ julia> Ellipsoidal([1.0,0.5], [π/2])
 julia> Ellipsoidal([1.0,0.5,0.5], [π/2,0.0,0.0])
 ```
 """
+
+# Right-hand rule; motion defined looking to the negative direction of the axis
+struct RotationRules
+  rot_seq::Symbol
+  motion::Vector{Symbol}
+  radian::Bool
+  main::Symbol
+  extrinsic::Bool
+end
+
+themes = Dict(
```

Comment:
Maybe rename this variable to `conventions`

Reply:
Or maybe `rules` so that when we access the key in the constructor it reads like `rules[convention]`.

Reply:
Done